cf/api/service_keys_test.go

```diff
@@ -142,6 +143,46 @@ var _ = Describe("Service Keys Repo", func() {
 		})
 	})
 
+	Describe("getting service key", func() {
```

Comment:
Sorry for being a pedant, but this might be better to be Describe("GetServiceKey()"), since the describe block is essentially for GetServiceKey()